backends/dpdk/dpdkUtils.cpp

```diff
@@ -75,6 +75,24 @@ bool isEightBitAligned(const IR::Expression *e) {
     return true;
 }
 
+bool isLargeFieldOperand(const IR::Expression *e) {
+    auto expr = e;
+    if (auto base = e->to<IR::Cast>()) expr = base->expr;
+    if (auto type = expr->type->to<IR::Type_Bits>()) {
```

Comment:
is slice expression expected here?

Reply:
Yes. We may get slice, but currently there is no support for  large operands for other operations such as shift and bitwise AND. Hence, we are not making any changes w.r.t that. Only simple assignments and casts are considered for now.